addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -403,6 +443,23 @@ private void updateChannelStates(LxControl control) {
                     updateState(channelId, OnOffType.OFF);
                 }
             }
+        } else if (control instanceof LxControlTimedSwitch) {
+            LxControlTimedSwitch timedSwitch = ((LxControlTimedSwitch) control);
+            Double value = timedSwitch.getState();
```

Comment:
Does it make sense that this returns a double and we only check for 1 or 0, there are some other types better suited for such things. What if it is 0.9999999999?

Reply:
@martinvw sure an int would work here, but I just did it similar to the LxControlSwitch snippet some lines above.

Reply:
That is basically what API documentation describes: double 1.0 value means on.

Reply:
Okay maybe we should then use an else for the second case or is that also described to be 0.0?

Reply:
Yes, 1 is on 0 is off and the state values are always double.